statick_tool/plugins/tool/xmllint_tool_plugin.py

```diff
@@ -21,9 +21,13 @@ def scan(self, package: Package, level: str) -> Optional[List[Issue]]:
         flags: List[str] = []
         flags += self.get_user_flags(level)
 
+        files: List[str] = []
```

Comment:
didn't these changes get merged in #403? how come they're still listed here

Reply:
I merged upstream (sscpac) main into my source branch and cleaned that up.